src/rtext.c

```diff
@@ -1069,13 +1069,14 @@ bool ExportFontAsCode(Font font, const char *fileName)
 // NOTE: Uses default font
 void DrawFPS(int posX, int posY)
 {
-    Color color = LIME;                         // Good FPS
-    int fps = GetFPS();
+    const int fps = GetFPS();
```

Comment:
What is the real benefit of `const` on this situation? I try to avoid `const` usage in raylib as much as possible.

Reply:
It is a hint to the compiler, and to the reader that author does not expect the value to change. Compiler can make optimizations based on this.

Reply:
Yes, just to keep the code simpler.

Reply:
Alright, next iter.